packages/datacommons-mcp/datacommons_mcp/server.py

```diff
@@ -418,19 +427,20 @@ async def search_indicators(
         - **Example 2: Child places of the World (Countries)**
           - **Call:**
             * `query="GDP"`
-            * `places=["World", "USA", "China", "Germany", "Nigeria", "Brazil"]`
+            * `parent_place="World"`
+            * `places=["USA", "China", "Germany", "Nigeria", "Brazil"]`
           - **Logic:**
-            1. Include the parent ("World").
+            1. Include the parent place ("World").
             2. Include 5-6 *diverse* child countries (e.g., from different continents, different economies).
-            3. This sampling helps discover the correct indicator DCID used for the `Country` place type, which you can then use in other tools (like `fetch_data` with a parent=`World` and child_type=`Country`).
+            3. This sampling helps discover the correct indicator DCID used for the `Country` place type, which you can then use in other tools (like `get_observations` with the parent's DCID in the `place_dcid` parameter and `child_place_type='Country'`).
 
         - **Example 3: Administrative Level Sampling**
 
           - **Goal:** Check data availability for different administrative levels (e.g., "population of US cities" vs "population of US states").
 
           - **Call:**
-            * **For Cities:** `query="population"`, `places=["USA", "New York City, USA", "Los Angeles, USA", "Chicago, USA", "Houston, USA", "Phoenix, USA"]`
-            * **For States:** `query="population"`, `places=["USA", "California, USA", "Texas, USA", "Florida, USA", "New York State, USA", "Pennsylvania, USA"]`
+            * **For Cities:** `query="population"`, `parent_place="USA"`, `places=["New York City, USA", "Los Angeles, USA", "Chicago, USA", "Houston, USA", "Phoenix, USA"]`
```

Comment:
nit: would it make sense for this example list to be the format <city>,<state> instead of <city>,USA (ie New York City, NY)? or is USA good?
If the maps api works well with both formats, then we can leave as is.